man/IDateTime.Rd

```diff
@@ -113,6 +113,7 @@ yearqtr(x)
   the S3 generic.}
   \item{units}{one of the units listed for truncating. May be abbreviated.}
   \item{ms}{ For \code{as.ITime} methods, what should be done with sub-second fractions of input? Valid values are \code{'truncate'} (floor), \code{'nearest'} (round), and \code{'ceil'} (ceiling). See Details. }
+  \item{label}{ for \code{wday} and \code{month} when set to true will return day and month names instead of integers, names will be in the regional language according to locale setting.}
```

Comment:
'for \code{wday} and \code{month} when set to true will return day and month names instead of integers' - Needs rephrasing (also, 'names will be in the regional language according to locale setting.' should be a separate sentence)

Reply:
change it to " \item{label}{ For \code{wday} and \code{month}, setting \code{label} to \code{TRUE} will return day and month names instead of integers. Names will be in the regional language according to the locale setting. }" is this good enough ?